server/src/main/java/io/unitycatalog/server/persist/dao/IdentifiableDAO.java

```diff
@@ -0,0 +1,27 @@
+package io.unitycatalog.server.persist.dao;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Id;
+import jakarta.persistence.MappedSuperclass;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.experimental.SuperBuilder;
+
+import java.util.UUID;
+
+@MappedSuperclass
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@SuperBuilder
+public class IdentifiableDAO {
+    @Id
+    @Column(name = "id", columnDefinition = "BINARY(16)")
```

Comment:
`columnDefinition` might be incompatible with mysql/postgre change in #211

Reply:
updated from main